etcdmain/etcd.go

```diff
@@ -520,6 +520,8 @@ func identifyDataDirOrDie(dir string) dirType {
 func setupLogging(cfg *config) {
 	capnslog.SetGlobalLogLevel(capnslog.INFO)
 	if cfg.debug {
+		formatter := capnslog.NewPrettyFormatter(os.Stdout, true)
```

Comment:
Nope, all logging should go to stderr.

On Fri, Nov 6, 2015 at 6:50 PM, Shaofeng Chen <notifications@github.com>
wrote:

> In etcdmain/etcd.go
> <https://github.com/coreos/etcd/pull/3820#discussion_r44165947>:
>
> > @@ -520,6 +520,8 @@ func identifyDataDirOrDie(dir string) dirType {
> >  func setupLogging(cfg *config) {
> >  	capnslog.SetGlobalLogLevel(capnslog.INFO)
> >  	if cfg.debug {
> > +		formatter := capnslog.NewPrettyFormatter(os.Stdout, true)
>
> Stderr is used by capnslog's default formatter. Thought Stdout is more
> appropriate for log output. Not quite sure.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/etcd/pull/3820/files#r44165947>.
>